src/Microsoft.Win32.SystemEvents/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,25 @@
+wtsapi32.dll!WTSRegisterSessionNotification
+wtsapi32.dll!WTSUnRegisterSessionNotification
+user32.dll!CreateWindowEx
+user32.dll!DefWindowProc
+user32.dll!DestroyWindow
+user32.dll!DispatchMessage
+user32.dll!GetClassInfo
+user32.dll!GetProcessWindowStation
+user32.dll!GetUserObjectInformation
+user32.dll!GetWindowThreadProcessId
+user32.dll!IsWindow
+user32.dll!KillTimer
+user32.dll!MsgWaitForMultipleObjectsEx
+user32.dll!PeekMessage
+user32.dll!RegisterClass
+user32.dll!RegisterWindowMessage
+user32.dll!PostMessageW
+user32.dll!SendMessage
+user32.dll!SetClassLongPtr
+user32.dll!SetTimer
+user32.dll!SetWindowLongPtr
+user32.dll!TranslateMessage
+user32.dll!UnregisterClass
+kernel32.dll!GetModuleHandleW
```

Comment:
Remove GetModuleHandle and LoadLibrary as they are in OneCore.

Reply:
They're required because I'm building the impl as netstandard and UAP doesn't support these.

Reply:
This can go away when I retarget to netcoreapp.